gms/impl/src/main/resources/mlModelESSearchQueryTemplate.json

```diff
@@ -0,0 +1,119 @@
+{
+  "function_score": {
+    "query": {
+      "bool": {
+        "should": [
+          {
+            "query_string": {
```

Comment:
looks like you want the ability to search over multiple fields with different boost factors? You can combine the different `should` clauses, similar to https://github.com/linkedin/datahub/blob/4cc6dd92b2dcb62e18717eff6699ce95896c7ad3/gms/impl/src/main/resources/corpGroupESSearchQueryTemplate.json ?
something like
`{
  "query_string": {
    "query": "$INPUT",
    "fields": [
      "evaluationDatasets^0.065",
      "trainingDatasets^0.075",
      "description^0.035",
      ...
    ],
    "default_operator": "OR",
    "analyzer": "whitespace_lowercase"
  }
}`

Reply:
Sure, I was just following the template for Dataset. I tend to like the more verbose version myself since I think it's more clear about what's going on to someone who is less familiar with ElasticSearch, but happy to consolidate it into the shorthand.